Directory.Build.targets

```diff
@@ -90,6 +90,7 @@
     <Move SourceFiles="$(PublishDir)clientxna.pdb" DestinationFolder="$(PublishDir)../../" Condition="Exists('$(PublishDir)clientxna.pdb')" />
   </Target>
 
+  <!-- Note: only the debug build should be started with the debugger, release builds will fail with a `Could not find Resource directory' error -->
```

Comment:
This is not LaTeX :)


`Could not find Resource directory' -> 'Could not find Resource directory'